### ISSUE TITLE
Can't find you on discord

### ISSUE THREAD

**Issue by Ox1dan** [Hyperclaw79/PokeBall-SelfBot#111]
I can't seem to DM you on discord. Is Hyperclaw79#3570 still how to DM you to donate for premium?

**Comment by Hyperclaw79** [comment]
Yes, it's the same username. Make sure you are not copying any extra space or the `@` by mistake.

**Comment by ShadowFox77** [comment]
um hey I am having issues getting this bot to connect to my discord when I try to run my script I get this https://ibb.co/g4C2kz

**Comment by Hyperclaw79** [comment]
Python is not added to your path. Go to Start→Remove/Change Programs. Right-click on python and choose Modify. This time, make sure `Add Python to Path` and `Add Python to Environment Variable` are enabled and reinstall.
